polars/polars-lazy/polars-plan/src/dsl/expr.rs

```diff
@@ -183,6 +195,10 @@ impl GetOutput {
         Default::default()
     }
 
+    pub fn infer_type() -> Self {
```

Comment:
This is against polars rationale. Polars doesn't infer types at runtime (execution time of plan), but needs to know them statically before the plan is executed. 

If we infer, other parts of the plan will fail.